client/omnichannel/directory/ChatTab.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import { useDebouncedValue, useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import { Table, Tag, Box } from '@rocket.chat/fuselage';
+import moment from 'moment';
+import { Meteor } from 'meteor/meteor';
+import { FlowRouter } from 'meteor/kadira:flow-router';
+
+import { useTranslation } from '../../contexts/TranslationContext';
+import { useEndpointDataExperimental } from '../../hooks/useEndpointDataExperimental';
+import GenericTable from '../../components/GenericTable';
+import FilterByText from '../../components/FilterByText';
+
+const useQuery = ({ text, itemsPerPage, current }, [column, direction], userIdLoggedIn) => useMemo(() => ({
+	sort: JSON.stringify({ [column]: direction === 'asc' ? 1 : -1 }),
+	open: false,
+	roomName: text,
+	agents: [userIdLoggedIn],
+	...itemsPerPage && { count: itemsPerPage },
+	...current && { offset: current },
+}), [column, current, direction, itemsPerPage, userIdLoggedIn, text]);
+
+const ChatTable = () => {
+	const [params, setParams] = useState({ text: '', current: 0, itemsPerPage: 25 });
+	const [sort, setSort] = useState(['closedAt', 'desc']);
+	const t = useTranslation();
+	const debouncedParams = useDebouncedValue(params, 500);
+	const debouncedSort = useDebouncedValue(sort, 500);
+	const userIdLoggedIn = Meteor.userId();
+	const query = useQuery(debouncedParams, debouncedSort, userIdLoggedIn);
+
+	const onHeaderClick = useMutableCallback((id) => {
+		const [sortBy, sortDirection] = sort;
+
+		if (sortBy === id) {
+			setSort([id, sortDirection === 'asc' ? 'desc' : 'asc']);
+			return;
+		}
+		setSort([id, 'asc']);
+	});
+
+	const onRowClick = useMutableCallback((_id) => {
+		FlowRouter.go('live', { id: _id });
+		// routing this way causes a 404 that only goes away with a refresh, need to fix in review
+		// livechatRoomRoute.push({ id: _id });
+	});
+
+	const { data } = useEndpointDataExperimental('livechat/rooms', query) || {};
```

Comment:
This call is returning nothing when the user is just a `livechat agent`, I mean, when the user is a `livechat manager` the endpoint is working properly.